src/dialects/mssql/query-generator-typescript.ts

```diff
@@ -83,4 +83,69 @@ export class MsSqlQueryGeneratorTypeScript extends AbstractQueryGenerator {
       this.quoteTable(tableName),
     ]);
   }
+
+  /**
+   * Generates an SQL query that returns all foreign keys of a table.
+   *
+   * @param   tableName   The table or associated model.
+   * @param   catalogName The catalog.
+   * @returns             The generated SQL query.
+   */
+  // TODO: is catalogName really required? If so, tests should fail if not provided
+  getForeignKeysQuery(tableName: TableNameOrModel, catalogName: string) {
```

Comment:
It's interesting that this doesn't fail when `catalogName` is not provided. I was hoping TS would pick this up